### ISSUE TITLE
The new WSD scheduler seems to have an issue.

### ISSUE THREAD

**Issue by gesen2egee** [kohya-ss/sd-scripts#1653]
![image](https://github.com/user-attachments/assets/92e76cfe-752f-4ae0-8669-72700508e8cb)

As expected, resuming from state should continue with the decayed learning rate. However, no matter how I try, it always sticks to the initial lr setting.

I used the following command:

--lr_scheduler warmup_stable_decay --lr_warmup_steps 0.1 --lr_scheduler_min_lr_ratio 0.1 --lr_decay_steps 0.2
